### PR TITLE
Addresses Testing and Docstring Cleanup

### PR DESCRIPTION
**Pull request by bcwaldon**
- adding tests for listing addresses 
- removing cruft from docstrings

### CODE REVIEW

File: stacktester/tests/test_server_addresses.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+        created_server = self.os.nova.create_server(server)
+        self.server_id = created_server['id']
+        self.os.nova.wait_for_server_status(self.server_id, 'ACTIVE')
+
+    def tearDown(self):
+        self.os.nova.delete_server(self.server_id)
+
+    def test_list_addresses(self):
```
Comment (dwalleck) [inline stacktester/tests/test_server_addresses.py:33]: Did someone recently add this functionality to 1.1? When I checked late last week, this didn't work

File: stacktester/tests/test_flavors.py [hunk 2]
```diff
@@ -71,9 +71,7 @@ class FlavorsTest(unittest.TestCase):
         #self.assertEqual(actual_links, expected_links)
 
     def test_show_flavor(self):
-        """
-        Verify the expected details are returned for a flavor
-        """
+        """Retrieve a single flavor"""
```
Comment (dwalleck) [inline stacktester/tests/test_flavors.py:74]: Is there a reason we changed the format of all the docstrings? This is shorter, but my initial thought was that these should state what the goal of the test was

Comment (bcwaldon) [inline stacktester/tests/test_flavors.py:74]: The refined docstrings still communicate the same information, just in a simpler form. I hated seeing "Verify the..." in every test I ran. What else is a test supposed to do than verify things are a certain way?

File: stacktester/tests/test_server_addresses.py (hunk 1 above)
Comment (bcwaldon) [inline stacktester/tests/test_server_addresses.py:33]: Note that each of the asserts is actually commented out with KNOWN-ISSUEs. I actually did some work today to fix it all in Nova, the merge prop just hasn't been accepted yet.

**Comment by throughnothing** [comment]
Looks like a good overall cleanup to me.  I'm going to trust you on the server adress tests, and since everything is commented out, I assume they'll be improved/fixed in the future if there are any issues once the functionality can be properly tested.

File: stacktester/tests/test_flavors.py (hunk 2 above)
Comment (dwalleck) [inline stacktester/tests/test_flavors.py:74]: For now for simple test cases that will work, but more complex scenarios may have more details. Tomato, tomato I suppose

**Pull request merged**